src/main/java/dev/morling/onebrc/CalculateAverage_royvanrijn.java

```diff
@@ -100,18 +100,18 @@ private void run() throws Exception {
             try (var fileChannel = (FileChannel) Files.newByteChannel(Path.of(FILE), StandardOpenOption.READ)) {
                 var bb = fileChannel.map(FileChannel.MapMode.READ_ONLY, segment.start(), segmentEnd - segment.start());
                 var buffer = new byte[64];
+                var pointerAndHash = new int[2];
 
-                // Force little endian:
-                bb.order(ByteOrder.LITTLE_ENDIAN);
```

Comment:
I guess that "forcing" doesn't work if the underlying datastructure (files) isn't something simple like a byte[].

I've written some tests locally that call these methods with BIG and LITTLE endian byte[] buffers to test, they both seem to work fine now.

Reply:
If a problem still arises, perhaps @gunnarmorling could run the "runTests()" method and see if something fails there... there shouldn't be any output.